packages/server/src/core/index.ts

```diff
@@ -15,6 +15,7 @@ export type {
   ProcedureArgs,
   ProcedureOptions,
 } from './procedure';
+export { inferParser } from './parser';
```

Comment:
Fails CI, might be a typo or export from the wrong place?

Reply:
Whoops, I fixed it to only export the type.